src/Input/Item.hs

```diff
@@ -178,7 +178,9 @@ hseToSig = tyForall
             TVar a b -> TVar a (b ++ [ty y])
         ty (TyVar _ x) = TVar (fromName x) []
         ty (TyCon _ x) = TCon (fromQName x) []
+#if MIN_VERSION_haskell_src_exts(1,20,0)
```

Comment:
See https://github.com/ndmitchell/neil#quirks - "Where I do use CPP, I make sure the code still loads in ghci alone, if all MIN_VERSION_foo macros have not been defined." - so you probably want #if !defined(MIN_VERSION_haskell_src_exts) || ...` so in `ghci` it assumes the newest version.